asyncify.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+wasm-opt --asyncify -O4 \
+  --pass-arg asyncify-import@env.compute_proof_and_propagate \
+  --pass-arg asyncify-import@env.request_stct_proof \
```

Comment:
As the [article](https://web.dev/asyncify/) says:

> Pass --asyncify to enable the transform, and then use --pass-arg=… to provide a comma-separated list of asynchronous functions, where the program state should be suspended and later resumed.

I did it locally (I do not remember if it actually works also with space separated) and it was working during my tests.